Client/Frontend/Reader/Reader.html

```diff
@@ -15,20 +15,14 @@
 <body data-readerStyle='%READER-STYLE%'>
   <div id="reader-header" class="header">
     <h1 id="reader-title"></h1>
-    <div id="reader-credits" class="credits">%READER-CREDITS%</div>
+    <div id="reader-credits" class="credits"></div>
     <script type="text/javascript" src="/reader-mode/javascript/Reader.js"></script>
-    <script nonce="%READER-TITLE-NONCE%">
-        document.getElementById("reader-page-title").textContent = "%READER-TITLE%";
-        document.getElementById("reader-title").textContent = "%READER-TITLE%";
-    </script>
   </div>
 
   <div id="reader-content" class="content">
-    %READER-CONTENT%
```

Comment:
note that this fix may break some reader mode pages if `READER-CONTENT` is actually supposed to contain HTML